content/docs/dvclive/live/make_stage_template.md

```diff
@@ -0,0 +1,69 @@
+# Live.make_stage_template()
+
+Writes template for a DVC pipeline [stage], capturing the outputs tracked by
+DVCLive, which you can use to [run] DVCLive experiments with DVC.
+
+```py
+def make_stage_template()
+```
+
+## Usage
+
+```py
+from PIL import Image
+from pathlib import Path
+from dvclive import Live
+
+live = Live(cache_images=True)
```

Comment:
have we stop pushing for context manager in all examples?

Reply:
I thought here, like in `make_dvcyaml()`, it was left out intentionally to show how it works without context manager/`end()`. No strong opinion.

Reply:
Makes sense. No strong opinion